### ISSUE TITLE
Update 'Join us' and other membership pages -- Summer

### ISSUE THREAD

**Issue by ryscher** [CDL-Dryad/dryad-product-roadmap#2396]
Updates associated with changing membership structure. Ask Sarah for text.

**Comment by ahamelers** [comment]
@sarahklippincott, Is this available/ready (for release May 23)

**Comment by sarahklippincott** [comment]
@ahamelers Thanks for checking. This won't be available until the June 6 release.

**Comment by ahamelers** [comment]
Thanks @sarahklippincott , I'll update the sprint/due date for the ticket

**Comment by ryscher** [comment]
Moved to next sprint, since text will not be finalized in this sprint.

**Comment by ryscher** [comment]
@sarahklippincott I'm assuming these changes are not yet ready, as  I still see many unresolved comments in the documents. I'm bumping the ticket to the next sprint...

**Comment by sarahklippincott** [comment]
That's right. Thanks @ryscher

**Comment by ahamelers** [comment]
@sarahklippincott Are changes ready to go up? Apologies, but I'm also not clear on how to translate [the linked document](https://docs.google.com/document/d/1epACR0Dls0dwWyJm7V_hZ8kPuxMRkNHmCkDzsTl9pCU/edit?usp=sharing) to specific web pages. Do all the pages that need content changes exist, or do some need to be created? Thanks!

**Comment by sarahklippincott** [comment]
@ahamelers We're not quite ready for these. Some new pages will need to be created--I'll outline this more clearly!

**Comment by ahamelers** [comment]
https://trello.com/c/htQclKsn/2-webpage-additions

**Comment by ahamelers** [comment]
Waiting for finalized text confirmation.

**Comment by ahamelers** [comment]
@sarahklippincott I have drafted up 4 of the new pages, since they did not have any outstanding comments on their google docs. https://github.com/CDL-Dryad/dryad-app/compare/main...membership-pages

Waiting on your go ahead for https://docs.google.com/document/d/1wOPlUMz5O_higmJ5ZGJrpb47YgkbOSKMbT-T-urrGOY/edit and feedback on the general help page restructuring for https://docs.google.com/document/d/1yr8zbsJVTqvBL448SseoSQXb-KvUNOV6J89C6u2i5Xw/edit#heading=h.cnuq7tg5gd8f

**Comment by ahamelers** [comment]
Sarah has said the terms page is ready. I removed the link in that page to the last remaining page, the researcher benefits and fees—it is not clear when that page will be complete. I will close this issue and when the content from that page needs to be added, I suggest we make a new issue.
